erpnext/patches/v8_1/remove_quotations_route_in_sidebar.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2017, Frappe and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+	frappe.reload_doctype("Portal Settings")
+
+	frappe.db.sql("""
+		delete from
+			`tabPortal Menu Item`
+		where
+			route = '/quotations' and title = 'Supplier Quotation'""")
+
+	frappe.db.sql("""
```

Comment:
@creamdory,
instead of using two `frappe.db.sql` use the single `frappe.db.sql`